### ISSUE TITLE
Published lib has differences with code source

### ISSUE THREAD

**Issue by Wenqer** [AscentSoftware/select-ts#1]
For a start - thanks for your work! That lib looks really promising and useful =)

Now to the issue, it looks like the version in npm 0.0.0 was published from different code sources.
For example, in GitHub sources, https://github.com/AscentSoftware/select-ts/blob/master/src/cond.ts#L18 cond operator are allowed to be used with any Kind1 HKT, but in npm, https://unpkg.com/select-ts@0.0.0/lib/cond.d.ts allowed to be used only on "Ord" | "Option" | "ReadonlyRecord" | "Eq" | "ReadonlyNonEmptyArray".

Such behavior makes impossible to use code example from the readme:
```
const condS = cond(getSelectM(task)); 
// Argument of type 'Selective1<"Task">' is not assignable to parameter of type 'Selective1<"Ord" | "Option" | "ReadonlyRecord" | "Eq" | "ReadonlyNonEmptyArray">'
```

**Comment by mstn** [comment]
You are right! Sorry. I will publish a new version soon!

**Comment by mstn** [comment]
@Wenqer I updated the code on NPM. Let me know if it works. I will try myself later.

**Comment by Wenqer** [comment]
@mstn cond function in 0.0.1 version still allowed to be used only on `"Ord" | "Option" | "ReadonlyRecord" | "Eq" | "ReadonlyNonEmptyArray"`. You can check it here https://unpkg.com/select-ts@0.0.1/lib/cond.d.ts

**Comment by mstn** [comment]
@Wenqer are you using `fp-ts@^2.5.3`?

**Comment by Wenqer** [comment]
@mstn Yep, you can check it here https://codesandbox.io/s/zealous-knuth-kwjec

**Comment by mstn** [comment]
@Wenqer if I add `import 'fp-ts';` at the top of `Selective.ts` file, `URIS` is populated with the right names in the export. Without it I guess that `URItoKind` is not augmented properly. Tbh I do not know why. Do you know if there is a better way than adding `import 'fp.ts'`?

**Comment by mstn** [comment]
@Wenqer sorry for the late reply. I should have fixed the problem with `select-ts@0.0.2`. Here, you can find an explanation https://github.com/gcanti/fp-ts/issues/1282

I'll close the issue. Feel free to reopen it if you have any problem.
